packages/broker/src/broker.ts

```diff
@@ -20,7 +19,7 @@ const logger = new Logger(module)
 
 export interface Broker {
     getNeighbors: () => readonly string[]
-    getStreams: () => readonly string[]
+    getSPIDs: () => Iterable<Protocol.SPID>
```

Comment:
I wonder if we should refer to them as `StreamPartitions` in names?
```suggestion
    getStreamPartitions: () => Iterable<Protocol.SPID>
```